src/main/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedProcessor.java

```diff
@@ -58,7 +61,16 @@ public void processQuestionnaireLinked(ResponseManagementEvent questionnaireLink
     UacQidLink uacQidLink = uacQidLinkOpt.get();
     Case caze = caseOpt.get();
 
+    // If already receipted
```

Comment:
Maybe beef up the comment here, because it's confusing as to why we wouldn't just check the `caze.isReceiptReceived` and then set it true if it it's false

Reply:
done